### ISSUE TITLE
Decreased circulating creatine kinase concentration

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7202]
**Preferred term label:**
Decreased circulating creatine kinase concentration

**Synonyms**

**Definition (free text, please give PubMed ID)**
An reduction in the level of the enzyme creatine kinase (also known as creatine phosphokinase, CPK; EC 2.7.3.2) in the blood. 
PMID: 31536231  PMID: 20460152
**Parent term (use hpo.jax.org/app)**
Abnormal circulating creatine kinase concentration HP:0040081

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added term as HP:0025659
